### ISSUE TITLE
Enable force update option for stream update

### ISSUE THREAD

**Issue by ilayaperumalg** [spring-cloud/spring-cloud-dataflow#4059]
It might be handy to have `stream update` forced. Currently, the update can only happen when there are properties difference between current and the previous deployment.

**Comment by sabbyanandan** [comment]
We already support it, though ([docs](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-stream-lifecycle-force-update)).

**Comment by ilayaperumalg** [comment]
Yes, I am aware of that option at the backend but what I meant is to get that in the dashboard. Should have created at the UI. Moving there.
